.jenkins/pytorch/test.sh

```diff
@@ -283,6 +284,12 @@ test_libtorch() {
   fi
 }
 
+test_aot_model_compiler() {
```

Comment:
nit: there are already "test_tensorexpr", "test_mobile_nnc" and "test_aot_model_compiler". I think it's better to consolidate the naming.